source/extensions/filters/http/ip_tagging/ip_tagging_filter.h

```diff
@@ -71,6 +73,7 @@ class IpTaggingFilterConfig {
   const Stats::StatName total_;
   const Stats::StatName unknown_tag_;
   std::unique_ptr<Network::LcTrie::LcTrie<std::string>> trie_;
+  const Http::LowerCaseString header_;
```

Comment:
nit: rename to `alt_header_` or something `ip_tag_header_` to make it more readable.